README.md

```diff
@@ -0,0 +1,54 @@
+# WeBloc
+
+> WeBloc is an open-source, community-driven event finder built on the MEAN Stack. Never has it been easier to find local events within your community.
+
+## Team
+
+  - __Product Owner__: Mark Keith
+  - __Scrum Master__: Trevor Pace
+  - __Development Team Members__: Nassir Zeinu
+
+## Table of Contents
+
+1. [Usage](#Usage)
+1. [Requirements](#requirements)
+1. [Development](#development)
+    1. [Installing Dependencies](#installing-dependencies)
+    1. [Tasks](#tasks)

```

Comment:
No hyperlink... if we're not going to have tasks, we should get rid of this part of the TOC.